homeassistant/components/weather/__init__.py

```diff
@@ -110,6 +118,10 @@ def state_attributes(self):
         if attribution is not None:
             data[ATTR_WEATHER_ATTRIBUTION] = attribution
 
+        forecast = self.forecast
+        if forecast is not None:
+            data[ATTR_FORECAST] = forecast
```

Comment:
I've done this now inside of the open weathermap component. Do you think it should be done here as well (or rather, just here, not in the component)?